File: Loiane_Training/Java_Basico/Exer Aula 18_19/Ex15.java
public class Ex15 {
    public static void main(String[] args) {
        double[] a = { 1, 2, 3, 4, 5, 5, 5, 5, 6, 7, 8, 9, 10 };

        var quantityEven = 0;
        for (int i = 0; i < a.length; i++) {
            if (a[i] % 2 == 0) {
                quantityEven++;
            }
        }

        var oddPercent = quantityEven * 100 / a.length;
        var evenPercent = 100 - oddPercent;

        System.out.println("Porcentagem de numeros pares: " + oddPercent + "%");
        System.out.println("Porcentagem de numeros impares: " + evenPercent + "%");
    }
}

File: loiane-training-csharp/procedural-poo/e/Ex15.cs
using System;

class Ex15
{
    static void Main(string[] args)
    {
        double[] vetorA = new double[13] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 14 };

        int contadorImpar = 0;
        for (int i = 0; i < vetorA.GetLength(0); i++)
        {
            if (vetorA[i] % 2 == 1)
            {
                contadorImpar++;
            }
        }

        Console.Write("\nVetor A: ");
        for (int i = 0; i < vetorA.GetLength(0); i++)
        {
            Console.Write(vetorA[i] + " ");
        }

        double porcentagemDeImpares = contadorImpar * 100 / vetorA.GetLength(0);

        System.Console.WriteLine("\ntamanho: " + vetorA.GetLength(0));
        System.Console.WriteLine("quantidade impares: " + contadorImpar);

        Console.WriteLine("\nPorcentagem de valores pares no Vetor A: {0}%", 100 - porcentagemDeImpares);
        Console.WriteLine("Porcentagem de valores impares no Vetor A: {0}%", porcentagemDeImpares);
    }
}